ct-app/core/components/utils.py

```diff
@@ -112,7 +112,11 @@ def mergeTopologyPeersSubgraph(
 
             peer.safe_address = subgraph_entry.safe_address
             peer.safe_balance = subgraph_entry.wxHoprBalance
-            peer.safe_allowance = float(subgraph_entry.safe_allowance)
+
+            if subgraph_entry.safe_allowance:
```

Comment:
Will this not also cover the case of 0? So 0 changes to None?

Reply:
Fixed.